Cargo.lock

```diff
@@ -6204,9 +6204,9 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b22a693222d716a9587786f37ac3f6b4faedb5b80c23914e7303ff5a1d8016e9"
 dependencies = [
- "proc-macro2 1.0.40",
- "quote 1.0.20",
- "syn 1.0.98",
+ "proc-macro2 1.0.43",
```

Comment:
Not sure why we need to change this. Seems like the same one from the other PR. If a lockfile changed globally for `main`, the best would be to issue a separate PR.

Reply:
Not sure exactly what caused the change. If I remember right when I checked out the new branch it checked out into unstaged changes (the cargo.lock was updated with these values) despite main not having this issue?? Not sure if we need a new PR for a 3 line change that I'm not sure where it even came from